notebooks/adastra/data-owners/data-owners-presentation.ipynb

```diff
@@ -351,9 +351,10 @@
     "# run this cell\n",
     "def make_private_tensors(split):\n",
     "    data_subjects = DataSubjectList.from_series(split[\"patient_ids\"])\n",
+    "    l_min_val,l_max_val = get_label_min_max_vals(MY_DATASET_URL)\n",
     "    return (\n",
     "        sy.Tensor(split[\"images\"]).private(min_val=0, max_val=255, data_subjects=data_subjects),\n",
-    "        sy.Tensor(split[\"labels\"]).private(min_val=0, max_val=7, data_subjects=data_subjects)\n",
+    "        sy.Tensor(split[\"labels\"]).private(min_val=l_min_val, max_val=l_max_val, data_subjects=data_subjects)\n",
```

Comment:
@rasswanth-s let's not use a function, we explicitly want to show the participants what are the min and max values here.

Reply:
Sure , I will revert :+1: